src/statusBar/statusBar.ts

```diff
@@ -44,11 +45,13 @@ export async function pollServiceLevel(): Promise<void> {
   if (!statusBarData) {
     return;
   }
+
+
+  const state = await getState();
   if (ONPREM) {
+    statusBarData.text = state?.cloud_connection_health_status !== "Ok" ? "Cloud connection issue" : null;
```

Comment:
@yonip23 @yanir-codota we need to sync on the exact text we want here.

assumption is we only are dealing with the cloud connection errors. in future if this changes we can augment this, for now seems simple and solves the issue at hand